ACPs/196-precompile-secp256r1/README.md

```diff
@@ -0,0 +1,83 @@
+# ACP-196: Precompile for secp256r1 Curve Support
+
+| ACP | 196 |
+| :--- | :--- |
+| **Title** | Precompile for secp256r1 Curve Support |
+| **Author(s)** | [Santiago Cammi](https://github.com/scammi), [Arran Schlosberg](https://github.com/ARR4N)  |
+| **Status** | Proposed ([Discussion](https://github.com/avalanche-foundation/ACPs/discussions/XXX)) |
+| **Track** | Standards |
+
+## Abstract
+
+This proposal introduces a precompiled contract that performs signature verifications for the secp256r1 elliptic curve on Avalanche's C-Chain and Subnet-EVM chains. The precompile will be implemented at address `0x0000000000000000000000000000000000000100` and will enable native verification of P-256 signatures, significantly improving gas efficiency for biometric authentication systems, WebAuthn, and modern device-based signing mechanisms.
+
+## Motivation
+
+The secp256r1 (P-256) elliptic curve is the standard cryptographic curve used by modern device security systems, including Apple's Secure Enclave, Android Keystore, WebAuthn, and Passkeys. However, Avalanche currently only supports secp256k1 natively, forcing developers to use expensive Solidity-based verification that costs 200k-330k gas per signature verification.
```

Comment:
Where is the 200k-330k estimate from?

Reply:
There are a few smart contract based verifiers implemented, the following posts goes through each of the solutions and display how much each cost.

https://hackmd.io/@1ofB8klpQky-YoR5pmPXFQ/SJ0nuzD1T#Smart-Contract-Based-Verifiers

https://www.alchemy.com/blog/what-is-rip-7212#how-does-rip-7212-compare-to-other-solutions